app-i18n/fcitx-configtool/fcitx-configtool-5.1.6-r1.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 2023-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+MY_PN="fcitx5-configtool"
+
+inherit cmake unpacker
+
+DESCRIPTION="Configuration module for Fcitx"
+HOMEPAGE="https://fcitx-im.org/ https://github.com/fcitx/fcitx5-configtool"
+SRC_URI="https://download.fcitx-im.org/fcitx5/${MY_PN}/${MY_PN}-${PV}.tar.zst -> ${P}.tar.zst"
+
+S="${WORKDIR}/${MY_PN}-${PV}"
+LICENSE="GPL-2+"
+SLOT="5"
+KEYWORDS="~amd64"
+IUSE="kcm +config-qt +qt5 qt6 test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	>=app-i18n/fcitx-5.1.6:5
+	app-text/iso-codes
+	sys-devel/gettext
+	virtual/libintl
+	x11-libs/libX11
+	x11-libs/libxkbfile
+	x11-misc/xkeyboard-config
+	qt5? (
+		>=app-i18n/fcitx-qt-5.1.4:5[qt5,-onlyplugin]
+		dev-qt/qtconcurrent:5
+		dev-qt/qtcore:5
+		dev-qt/qtdbus:5
+		dev-qt/qtgui:5
+		dev-qt/qtsvg:5
+		dev-qt/qtwidgets:5
+		dev-qt/qtx11extras:5
+		kde-frameworks/kwidgetsaddons:5
+		config-qt? (
+			kde-frameworks/kitemviews:5
+		)
+	)
+	qt6? (
+		>=app-i18n/fcitx-qt-5.1.4:5[qt6,-onlyplugin]
+		dev-qt/qtbase:6[concurrent,dbus,gui,widgets]
+		dev-qt/qtsvg:6
+		kde-frameworks/kwidgetsaddons:6
+		config-qt? (
+			kde-frameworks/kitemviews:6
+		)
+	)
+	kcm? (
```

Comment:
I think it would be cleaner if you include the `kcm?` inside each of the qt variants above, so it looks like:

```
  !qt6? (
    ...
    kcm? (
      ...
    )
  )
  qt6? (
    ...
    kcm? (
      ...
    )
  )
  kcm? ( x11-libs/libxkbcommon )
```